scalableminds/webknossos
# Don't re-fill all available recommended settings to task type form

If there is already a non-null recommendation-configuration, don't add the defaults.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- edit the default task type and add a recommended configuration
- use the "remove" buttons to edit the recommended configuration
- save task type and re-open the edit page
- the recommended configuration should look exactly as before (i.e., no additional attributes should be added)
- disabling the recommendation, saving + reloading, re-enabling the recommendation should show all available properties again

### Issues:
- fixes #5270

------
